Modules/System/XML Validation/src/XmlValidationImpl.Codeunit.al

```diff
@@ -0,0 +1,69 @@
+codeunit 50101 "Xml Validation Impl."
+{
+    Access = Internal;
+
+    var
+        InvalidXmlErr: Label 'The xml definition is invalid.';
+        InvalidSchemaErr: Label 'The schema definition is not valid xml.';
+
+    procedure ValidateAgainstSchema(Xml: Text; XmlSchema: Text; Namespace: Text; var ErrorText: Text): Boolean
+    var
+        XmlDoc: XmlDocument;
+        XmlSchemaDoc: XmlDocument;
+    begin
+        if not XmlDocument.ReadFrom(Xml, XmlDoc) then
+            Error(InvalidXmlErr);
```

Comment:
This error isn't documented in `ValidateAgainstSchema`.

You can use the [error tag](https://github.com/microsoft/ALAppExtensions/wiki/AL-Code-Documentation#error_tag).

Reply:
Added error tag.